pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -196,13 +198,25 @@ private static boolean isAggregateExpression(Expression expression) {
     return false;
   }
 
-  public static Set<String> extractIdentifiers(List<Expression> expressions) {
+  /**
+   * Extract all the identifiers from given expressions.
+   *
+   * @param expressions
+   * @param excludeAs if true, ignores the right side identifier for AS function.
+   * @return all the identifier names.
+   */
+  public static Set<String> extractIdentifiers(List<Expression> expressions, boolean excludeAs) {
```

Comment:
May be a non-issue here, but just wanted to point out that there are aggregation functions that take multiple args, some of which are expected to be literals (even though they look like expressions, eg distinctCountThetaSketch).

Reply:
I see, if those args are single-quoted then it should be fine .